standalone-tools/wss-scan.sh

```diff
@@ -0,0 +1,121 @@
+###### Information ############################################################################
+# Name:          wss-scan.sh
+# Maintainer:    ODFE Infra Team
+# Language:      Shell
+#
+# About:         This script is to scan the odfe distros for vulnerabilities and licenses
+#                It will scan the repositories and send the WhiteSource link to the mail 
+#                of the user. 
+#
+# Prerequisites: Need to set the Java Path
+#                Need to set the recepient mail in wss-scan.config
+#
+# Usage:         ./wss-scan.sh
+#
+# Starting Date: 10-13-2020
+# Modified Date: 10-15-2020
+###############################################################################################
+
+#Download the WhiteSource Agent 
+curl -LJO -sS https://github.com/whitesource/unified-agent-distribution/releases/latest/download/wss-unified-agent.jar > /dev/null
+
+echo "Download completed"
+
+#Scanning the config file for the user configurations
+source wss-scan.config
+gitRepos=${gitRepos//,/$'\n'}  # change the semicolons to white space
```

Comment:
Then why not just use a double quoted string in config file, with whitespace separations at first?

Reply:
Is it semicolons or comma in this case, I got confused.